### DISCUSSION TITLE
run pulldown mode with multimeric bait protein

### DISCUSSION THREAD

**Discussion by rukibuki** [KosinskiLab/AlphaPulldown#348] [Q&A] [answered]
Hi, 
I might be missing something obvious, but how do I perform pulldown using a multimeric bait protein (homodimer specifically)? I can see multimeric stuff in the examples but it does not seem, to me at least, to be for performing a pulldown using a multimeric bait protein. I am sorry if it is just me misunderstanding stuff.

I should mention that AF2 and AF3 does not fold the right protein if I give them a single sequence containing both copies, they need two copies of the monomer to fold it.

**Comment by dingquanyu** [comment, marked as answer]
Hi @rukibuki 

Certainly you could use homo multimer as a bait protein. Please refer to this issue for the solution: #188 

Yours
Dingquan

**Reply by rukibuki** [reply]
ahh such a simple solution. I just tested it and it works. So how would I do if my bait protein was a hetero multimer? just two different bait files or what?
